drkafka/src/main/java/com/pinterest/doctorkafka/KafkaClusterManager.java

```diff
@@ -600,10 +600,21 @@ public UnderReplicatedReason getUnderReplicatedReason(String brokerHost,
     Map<TopicPartition, Integer[]> replicasMap = new HashMap<>();
     boolean success = true;
 
+    Map<String, PriorityQueue<KafkaBroker>> brokerQueueByLocality = null;
+    if(clusterConfig.enabledLocalityAwareness()){
+      brokerQueueByLocality = kafkaCluster.getBrokerQueueByLocality();
+    }
     PriorityQueue<KafkaBroker> brokerQueue = kafkaCluster.getBrokerQueue();
+
     for (OutOfSyncReplica oosReplica : outOfSyncReplicas) {
-      Map<Integer, KafkaBroker> replacedNodes =
-          kafkaCluster.getAlternativeBrokers(brokerQueue, oosReplica);
+      Map<Integer, KafkaBroker> replacedNodes;
+      if(clusterConfig.enabledLocalityAwareness()){
```

Comment:
how about use the following to save a few lines? 

        replacedNodes = clusterConfig.enabledLocalityAwareness() 
            ? kafkaCluster.getAlternativeBrokersByLocality(brokerQueueByLocality, oosReplica)
            : kafkaCluster.getAlternativeBrokers(brokerQueue, oosReplica);